cucumber-messages/ruby/Rakefile

```diff
@@ -1,6 +1,7 @@
 # encoding: utf-8
 require 'rubygems'
 require 'bundler'
+load 'protobuf/tasks/compile.rake'
```

Comment:
This file should not be modified. The build copies it from `.templates/ruby/Rakefile`. We may need some dynamic loading here, for examople load all files in `./rake/*.rb`